prog/weaveutil/expose.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	weavenet "github.com/weaveworks/weave/net"
+)
+
+func exposeNAT(args []string) error {
+	if len(args) < 1 {
+		cmdUsage("expose-nat", "<cidr>...")
+	}
+
+	cidrs, err := parseCIDRs(args[1:])
```

Comment:
You don't need the `[1:]` here - the command was already stripped off by weaveutil main